account_invoice_supplier_ref_reuse/__manifest__.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2018 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    'name': 'Reuse Supplier Invoice References',
+    'version': '11.0.1.0.0',
+    'summary': 'Makes it possible to reuse supplier invoice references',
+    'author': 'Camptocamp SA, '
```

Comment:
s/Camptocamp SA/Camptocamp/ (we'd like to have to have the same name everywhere for easier lookups in apps.odoo.com)